wearipedia/devices/oura/oura_ring3.py

```diff
@@ -0,0 +1,109 @@
+from datetime import datetime, time, timedelta
+
+from ...utils import bin_search, seed_everything
+from ..device import BaseDevice
+from .oura_ring3_authenticate import *
+from .oura_ring3_fetch import *
+from .oura_ring3_gen import *
+
+class_name = "Oura_Ring_3"
+
+
+class Oura_Ring_3(BaseDevice):
+    """This device allows you to work with data from the `Oura Ring 3  <(https://ouraring.com/)>`_ device.
+    Available datatypes for this device are:
+
+    * `sleep`: sleep data
+    * `daily_activity`: daily activity data
+    * `activity`: activity data
+    * `ideal_bedtime`: ideal bedtime data
+    * `readiness`: readiness data
+    * `heart_rate`: heart_rate
+
+    :param seed: random seed for synthetic data generation, defaults to 0
+    :type seed: int, optional
+    :param synthetic_start_date: start date for synthetic data generation, defaults to "2022-03-01"
+    :type synthetic_start_date: str, optional
+    :param synthetic_end_date: end date for synthetic data generation, defaults to "2022-06-17"
+    :type synthetic_end_date: str, optional
+    :param single_day: end date for real data data fetching, defaults to "2022-09-19"
```

Comment:
It doesn't look like the single_day parameter is set or used here. You can either remove it from the docstring here or include it in the device